auction-server/src/state.rs

```diff
@@ -1027,8 +1052,9 @@ impl Store {
             .collect();
         parsed_opps.map_err(|e| {
             tracing::error!(
-                "Failed to convert opportunity to OpportunityParamsWithMetadata: {}",
-                e
+                "Failed to convert opportunity to OpportunityParamsWithMetadata: {} - query: {:?}",
+                e,
+                query.sql(),
```

Comment:
this is not gonna include the actual parameters in the query but just $1, $2, etc. Not sure if this is actually what you wanted